ci/pipelines/carbon-relay-ng.yml

```diff
@@ -0,0 +1,243 @@
+            LABEL_release_number: ((.:release-number))
+            LABEL_release_name: ((.:release-name))
+            LABEL_release_sha: ((.:release-sha))
+          config:
+            platform: linux
+            image_resource:
+              type: registry-image
+              source:
+                repository: concourse/oci-build-task
+            inputs:
+              - name: carbon-relay-ng-src
+            outputs:
+              - name: image
+            run:
+              path: build
+        - task: create-tags-for-container
+          config:
+            platform: linux
+            image_resource:
+              type: registry-image
+              source:
+                repository: alpine
+            inputs:
+              - name: tags
+              - name: carbon-relay-ng-version
+            outputs:
+              - name: container-additional-tags
+            run:
+              path: ash
+              args:
+                - -eu
+                - -c
+                - |
+                  cat tags/tags | tee -a container-additional-tags/additional-tags-ecr container-additional-tags/additional-tags-dockerhub # Release tag
+                  cat carbon-relay-ng-version/version.txt | tee -a container-additional-tags/additional-tags-ecr container-additional-tags/additional-tags-dockerhub
+                  cat carbon-relay-ng-version/version.txt | cut -f 1 -d "-" | tee -a container-additional-tags/additional-tags-ecr container-additional-tags/additional-tags-dockerhub
+      - task: run-tests-against-container
+        privileged: true
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: govukpay/concourse-runner
+          inputs:
+            - name: image
+            - name: carbon-relay-ng-src
+          run:
+            path: /bin/bash
+            args:
+              - -c
+              - |
+                source /docker-helpers.sh
+                start_docker
+
+                set -euo pipefail
+
+                function cleanup {
+                  echo "CLEANUP TRIGGERED"
+                  clean_docker
+                  stop_docker
+                  echo "CLEANUP COMPLETE"
+                }
+
+                trap cleanup EXIT
+
+                echo "Loading docker image"
+                docker load -i image/image.tar
+                echo
+
+                echo "Tagging image with carbon-relay-ng:test"
+                docker tag $(cat image/digest) carbon-relay-ng:test
+                echo
+
+                cd carbon-relay-ng-src/govuk-tests
+                echo "Updating docker-compose.yml to use built image"
+                sed -i -E 's/build: ".*"/image: "carbon-relay-ng:test"/' docker-compose.yml
+                echo
+
+                echo "Running tests"
+                docker-compose up --exit-code-from test-receiver
+                echo "Tests passed"
```

Comment:
Do we want any Slack notifications if the jobs fail?

Reply:
Oh yeah, I think we definitely do

Reply:
I'll set the usual ones up for pass/fail